docs/learn/getting-started-boss-room.md

```diff
@@ -25,9 +25,9 @@ Using Windows' built-in extracting tool may generate an "Error 0x80010135: Path
 - You are now ready to add the project to Unity Hub.
 
 :::important Compatibility
-- Boss Room supports those platforms supported by MLAPI (Windows and Mac).
+- Boss Room supports those platforms supported by MLAPI (Windows, MacOS,Ubuntu 20.4 LTS, Ubuntu 18.04 LTS and CentOS 7).
```

Comment:
Minor but missing space between `MacOS,` and `Ubuntu`

Reply:
(same formatting issue present in the other instances)

Reply:
Thanks should be fixed now